src/core/utils/modal.tsx

```diff
@@ -45,9 +45,9 @@ function Modal({
 
   return (
     <div
-      className={`modal ${
-        modalIds.includes(modalId) ? "modal-show" : ""
-      } ${additionalClasses}`}
+      className={`modal ${modalIds.includes(modalId) ? "modal-show" : ""} ${
```

Comment:
Great modification. Consider using the `csx` library for the class names. It is easier to read I think.